### PR TITLE
gpsbabel: update to 1.7.0

### PR DESCRIPTION
**Pull request by tsteven4**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 11.2.3 20D91
Xcode 12.4 12D4e

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: textproc/gpsbabel/Portfile [hunk 1]
```diff
@@ -26,63 +21,57 @@ long_description    GPSBabel converts GPS waypoints, tracks, and routes \
...
-
-    configure.universal_args-delete \
-        --disable-dependency-tracking
+    # configure with qmake
+    # see https://github.com/GPSBabel/gpsbabel/pull/611
+    patchfiles-append patch-pr611.diff
+    patch.pre_args -p1
 
     depends_lib-append  \
+        port:libusb     \
```
Comment (mf2k) [inline textproc/gpsbabel/Portfile:38]: Other ports allow other versions of libusb to be used. Can you test and use this instead?

`path:lib/pkgconfig/libusb-1.0.pc:libusb
`

```
$ port list libusb*
libusb                         @1.0.24         devel/libusb
libusb-compat                  @0.1.7          devel/libusb-compat
libusb-devel                   @20210217-d97e85ef devel/libusb
libusb-legacy                  @0.1.12         devel/libusb-legacy
```

**Review by mf2k — CHANGES_REQUESTED** [review]

File: textproc/gpsbabel/Portfile (hunk 1 above)
Comment (tsteven4) [reply textproc/gpsbabel/Portfile:38]: Your suggestion allows libusb (1.0.24) or libusb-devel (20210217-d97e85ef) to be used.  I'll push the change.

Comment (tsteven4) [reply textproc/gpsbabel/Portfile]: Note I also added "qt5.min_version         5.9", which is the minimum version of Qt that gpsbabel 1.7.0 requires.

**Review by mf2k — APPROVED** [review]

File: textproc/gpsbabel/Portfile [hunk 2]
```diff
@@ -26,63 +21,59 @@ long_description    GPSBabel converts GPS waypoints, tracks, and routes \
 platforms           darwin
 homepage            https://www.gpsbabel.org
 
-depends_lib-append  port:expat \
-                    port:libusb-compat
+checksums           rmd160  dd41329723a41749a6ef929e0766fba708509116 \
+                    sha256  a5ef3b1744488d86d438291df2d65135cb04c435680fe7f64f5a94a3814ebea4 \
+                    size 16720596
```
Comment (ryandesign) [inline textproc/gpsbabel/Portfile:26]: Can the size be intended to match the other checksum types, please?
```suggestion
                    size    16720596
```

File: textproc/gpsbabel/files/makefile_macports_install [hunk 3]
```diff
@@ -3,9 +3,8 @@
...
 	$(LRELEASE) app.pro
-	mkdir -p objects/GPSBabelFE.app/Contents/MacOS/translations
-	cp *.qm  objects/gpsbabelFE.app/Contents/MacOS/translations
-	cp -r help/*.html  objects/GPSBabelFE.app/Contents/MacOS
-	cp gmapbase.html objects/GPSBabelFE.app/Contents/MacOS
-	cp -r objects/GPSBabelFE.app $(INSTALL_ROOT)$(APPLICATION_DIR)
+	mkdir -p GPSBabelFE.app/Contents/MacOS/translations
+	cp gpsbabelfe_*.qm  coretool/gpsbabel_*.qm GPSBabelFE.app/Contents/MacOS/translations
+	cp gmapbase.html GPSBabelFE.app/Contents/MacOS
+	cp -r GPSBabelFE.app $(INSTALL_ROOT)$(APPLICATION_DIR)
```
Comment (ryandesign) [inline textproc/gpsbabel/files/makefile_macports_install:9]: Per cp(1):

> Historic versions of the cp utility had a -r option.  This implementation supports that option; however, its use is strongly discouraged, as it does not correctly copy special files, symbolic links, or fifo's.

Therefore:

```suggestion
	cp -R GPSBabelFE.app $(INSTALL_ROOT)$(APPLICATION_DIR)
```

File: textproc/gpsbabel/Portfile (hunk 2 above)
Comment (tsteven4) [reply textproc/gpsbabel/Portfile]: done and pushed

File: textproc/gpsbabel/files/makefile_macports_install (hunk 3 above)
Comment (tsteven4) [reply textproc/gpsbabel/files/makefile_macports_install]: done and pushed

**Pull request merged**